src/python/dxpy/bindings/apollo/dataset.py

```diff
@@ -0,0 +1,48 @@
+from dxpy import DXHTTPRequest
+
+
+class Dataset:
```

Comment:
I'm now considering that calling it Dataset class might be confusing as the parameters passed here regard the record (because without visualize internal function all we have is the record-id). Do you guys think it's worth renaming?
Just small recap: 
- visualize gets info from record (regarding record, database and more)
- descriptor gets info from dataset regarding dataset